gazebo_worlds/package.xml

```diff
@@ -54,6 +54,7 @@
   <!-- The export tag contains other, unspecified, tags -->
   <export>
     <!-- Other tools can request additional information be placed here -->
-
+    <gazebo_ros gazebo_model_path="${prefix}/models/"/>
```

Comment:
Oh. I'm blind! You DO need to add the ` <exec_depend>gazebo_ros</exec_depend>` for this to work.